app/src/main/java/org/mozilla/focus/utils/Settings.kt

```diff
@@ -178,11 +178,11 @@ class Settings(
                 .commit()
         }
 
-    var shouldShowCfrForShieldToolbarIcon: Boolean
-        get() = preferences.getBoolean(getPreferenceKey(R.string.pref_cfr_visibility_for_shield_toolbar_icon), true)
```

Comment:
We agreed that I should keep the name with "tracking_protection" because this comes from AC . "ShieldToolbarIcon" is not a good name because the icon can change and after that the code will not reflect what is in the GUI.